Components/GroupModules/GroupModule_GroupPosition/Config.xml

```diff
@@ -229,6 +228,66 @@
 				</Scripts>
 			</Slider>
 
+			<Slider parentKey="SizeX" inherits="TellMeWhen_SliderTemplate">
+				<Anchors>
+					<Anchor point="TOP" relativeKey="$parent.Strata" relativePoint="BOTTOM"  y="-20"/>
+					<Anchor point="LEFT" relativeKey="$parent.Strata" />
+					<Anchor point="RIGHT" relativeKey="$parent.Strata" />
+					
+				</Anchors>
+				<Scripts>
+					<OnLoad>						
+						TMW:CInit(self)
+
+						self:SetTexts(TMW.L["UIPANEL_GROUP_SIZE_X"], TMW.L["UIPANEL_GROUP_SIZE_X_DESC"])
+						self:SetSetting("SizeX")
+
+
+						self:CScriptAdd("SettingTableRequested", function(requester)
+							return TMW.CI.group:GetSettingsPerView()
```

Comment:
I think this is the right way to get to gspv, but if there's a simpler way to access it I'll switch